librz/core/cmd/cmd_print.c

```diff
@@ -3152,15 +3152,16 @@ static bool core_disassembly(RzCore *core, int n_bytes, int n_instrs, RzCmdState
 		.cbytes = cbytes,
 	};
 
-	ut8 *buf = malloc(n_bytes + 1);
+	ut8 *buf = calloc(1, n_bytes + 1);
```

Comment:
```suggestion
	ut8 *buf = RZ_NEWS0(ut8, n_bytes + 1);
```

More a fan of `RZ_NEWS0`.
Mostly because we encourage new comers to use our macros and not `calloc`